src/NServiceBus.MessagingBridge/Utils/DeterministicGuid.cs

```diff
@@ -0,0 +1,43 @@
+﻿#nullable enable
+
+namespace NServiceBus.MessagingBridge;
+
+using System;
+using System.Buffers;
+using System.Security.Cryptography;
+using System.Text;
+
+static class DeterministicGuid
```

Comment:
If you got the code for this from somewhere, we usually link back to the source. Also pay attention to licenses that the original source code might have.

Reply:
Thanks, @dvdstelt. This was in NServiceBus.Core and didn't seem to be lifted from elsewhere.